tsconfig.json

```diff
@@ -4,11 +4,13 @@
         "target": "es6",
         "outDir": "out",
         "alwaysStrict": true,
+		"esModuleInterop": true,
         "lib": [
             "es6",
             "dom",
         ],
         "sourceMap": true,
+		"skipLibCheck": true,
```

Comment:
[F] We need to set this options in order to be able to adopt sinon-chai.

Reply:
question: what specifically about this causes typescript to dislike sinon-chai?

Reply:
The error is: 
```
node_modules/@types/sinon-chai/index.d.ts:107:31 - error TS2694: Namespace 'Chai' has no exported member 'ChaiPlugin'.

107 declare const sinonChai: Chai.ChaiPlugin;
```
Much googling reveals that `ChaiPlugin` (for some reason) is private within `Chai`. I will take a look to see if I can find another way around.

Reply:
very glad you asked about this, I just had to update the types and everything now works.

Reply:
Excellent hah!